### ISSUE TITLE
Cheat (Two-handed) should be renamed.

### ISSUE THREAD

**Issue by Shivaan88k** [fgsfdsfgs/perfect_dark#436]
Love the addition of the Two-Handed cheat, but I was honestly confused when I first saw it, before testing it. Maybe rename to Dual Wield Anything or something similar.

**Comment by fgsfdsfgs** [comment]
Yes, it is named that because of a slight limitation of how strings are handled in the game. This is the closest string to something proper that's in the game's language banks.

**Comment by blaiz** [comment]
I'm curious as well cause I want to rename "All Guns in Solo" to just "All Guns". Wondering how to tweak strings.
